src/Clipper.jl

```diff
@@ -991,15 +1015,21 @@ function Base.show(io::IO, p::__ClipperPath)
     if isempty(p)
         return
     end
+    print("Path([")
     for i = 1:length(p)-1
         print(io, string("(", x(p[i]), ",", y(p[i]), "), "))
     end
     print(io, string("(", x(p[end]), ",", y(p[end]), ")"))
+    print("])")
 end
 
 function Base.show(io::IO, p::__ClipperPaths)
-    for i = 1:length(p)
+    len = length(p)
+    for i = 1:len
         show(io, p[i])
+        if i < len
+            print(io, "; ")
+        end
```

Comment:
I like that you made `show(::Path)` output code that can construct it.  Can you do the same for `Paths`?